lib/pluginUtils.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const assert = require('assert')
+
+function getMeta (fn) {
+  return fn[Symbol.for('plugin-meta')]
+}
+
+function shouldSkipOverride (fn) {
+  return !!fn[Symbol.for('skip-override')]
+}
+
+function checkDependencies (fn) {
+  const meta = getMeta(fn)
```

Comment:
Is the `getMeta` function really necessary? I don't see it being used anywhere else.

Reply:
At the moment is only here, but once we add more utils to the plugin stuff we can use that function more.